test/controllers/api/v2/ping_controller_test.rb

```diff
@@ -31,4 +31,34 @@ class Api::V2::PingControllerTest < ActionController::TestCase
     results = ActiveSupport::JSON.decode(@response.body)
     assert_not results.empty?, 'Should response with statuses'
   end
+
+  test 'should include version headers if setting enabled and user logged in' do
+    Setting[:expose_version] = true
+    User.current = users(:admin)
+    Ping.stubs(:statuses).returns({})
+    get :statuses
+    assert_equal response.headers['Foreman_version'], SETTINGS[:version].full
+    assert_equal 'v2', response.headers['Foreman_api_version']
```

Comment:
A general suggestion is to only have a single assertion in unit tests. We also always return the API version so I'd drop this line.
```suggestion
```